reference/commands/misc/inspect.rst

```diff
@@ -68,3 +68,7 @@ If no specific attributes are defined via ``-a``, then, some default attributes
     apply_env: True
     build_policy: None
     topics: None
+    settings: ('os', 'arch', 'compiler', 'build_type')
+    options
+        shared: [True, False]
+    default_options: shared=False
```

Comment:
I would like to see this in the same shape of the options section

Reply:
I got from command inspect result. I could update, but won't reflect the real output.

Reply:
Done!